react-app/src/components/projects/AddProject.tsx

```diff
@@ -0,0 +1,375 @@
+            setLoadingOptions(false);
+            setFuzzySearchOptions([
+              ...response.Parcels.map((a) => ({ ...a, Type: 'Parcel' })),
+              ...response.Buildings.map((a) => ({ ...a, Type: 'Building' })),
+            ]);
+          });
+        }}
+        filterOptions={(options) => options.filter((x) => !rows.find((row) => row.Id === x.Id))}
+        value={autoCompleteVal}
+        renderInput={(params) => (
+          <TextField
+            {...params}
+            onBlur={() => setFuzzySearchOptions([])}
+            InputProps={{
+              ...params.InputProps,
+              startAdornment: (
+                <InputAdornment position="start">
+                  <Search />
+                </InputAdornment>
+              ),
+            }}
+            label={'Search by any keyword...'}
+          />
+        )}
+      />
+      <DataGrid
+        getRowId={(row) => row.Id + row.Type}
+        autoHeight
+        hideFooter
+        columns={columns}
+        rows={rows}
+      />
+    </Box>
+  );
+};
+
+const AddProject = () => {
+  const formMethods = useForm({
+    defaultValues: {
+      Name: '',
+      TierLevelId: null,
+      Notes: '',
+      Assessed: '',
+      NetBook: '',
+      Estimated: '',
+      Appraised: '',
+      Metadata: {
+        estimatedSalesCost: '',
+        estimatedProgramRecoveryFees: '',
+        surplusDeclaration: false,
+        tripleBottomLine: false,
+        applyForEnhancedExemption: false,
+      },
+      Approval: false,
+    },
+  });
+
+  const [showNoPropertiesError, setShowNoPropertiesError] = useState(false);
+  const [rows, setRows] = useState([]);
+  const theme = useTheme();
+  const api = usePimsApi();
+  const { data: tierData, loadOnce: loadTiers } = useDataLoader(api.lookup.getProjectTierLevels);
+  loadTiers();
+
+  return (
+    <Box
+      display={'flex'}
+      gap={'1rem'}
+      mt={'2rem'}
+      mb={'2rem'}
+      flexDirection={'column'}
+      width={'38rem'}
+      marginX={'auto'}
+      boxShadow={'2em'}
+    >
+      <Box>
+        <NavigateBackButton navigateBackTitle={'Back'} onBackClick={() => {}} />
+      </Box>
+      <Typography variant={'h2'} mb={'2rem'}>
+        Create disposal project
+      </Typography>
+      <FormProvider {...formMethods}>
+        <Typography variant="h5">Project information</Typography>
+        <Grid container spacing={2}>
+          <Grid item xs={12}>
+            <TextFormField required fullWidth name={'Name'} label={'Name'} />
+          </Grid>
+          <Grid item xs={12}>
+            <AutocompleteFormField
+              required
+              name={'TierLevelId'}
+              label={'Assign tier'}
+              options={tierData?.map((tier) => ({ value: tier.Id, label: tier.Name })) ?? []}
+            />
+          </Grid>
+          <Grid item xs={12}>
+            <TextFormField required fullWidth name={'Notes'} label={'Notes'} />
```

Comment:
I don't think Notes is a required field.
Also making it a bit bigger with multiline would be nice.
```suggestion
            <TextFormField multiline minRows={3} fullWidth name={'Notes'} label={'Notes'} />
```